src/FNA3D_Driver_OpenGL.c

```diff
@@ -1155,18 +1167,36 @@ void OPENGL_SetScissorRect(void* driverData, FNA3D_Rect *scissor)
 	}
 }
 
-void OPENGL_GetBlendFactor(
+FNA3D_Color OPENGL_GetBlendFactor(
 	void* driverData,
 	FNA3D_Color *blendFactor
 ) {
-	/* TODO */
+	OpenGLDevice *device = (OpenGLDevice*) driverData;
+	return device->blendColor;
```

Comment:
This should be an `SDL_memcpy(blendFactor, &device->blendColor, sizeof(FNA3D_Color);` (This is needed for easier C# interop)